packages/create-sitecore-jss/src/common/processes/transform.ts

```diff
@@ -61,16 +61,40 @@ export const merge = (targetObj: JsonObjectType, sourceObj: JsonObjectType): Jso
   return mergeObject(targetObj, sourceObj);
 };
 
-export const concatEnv = (targetContent: string, sourceContent: string): string => {
-  const env = parse(sourceContent);
-  if (
-    env &&
-    Object.keys(env).length > 0 &&
-    Object.keys(env).every((value) => targetContent.includes(value))
-  ) {
-    // Don't add if the target already contains every .env value
-    return targetContent;
-  }
+export const mergeEnv = (targetContent: string, sourceContent: string): string => {
+  const sourceEnv = parse(sourceContent);
+
+  if (!sourceEnv || !Object.keys(sourceEnv).length) return targetContent;
+
+  const targetEnv = parse(targetContent);
+
+  const sourceLines = sourceContent.split('\n');
```

Comment:
Shouldn't we split by '\r\n' here since we are enforcing this in the repo (in .gitattributes)?

Reply:
I think it shouldn't affect, since I join later by using '\n'. In any case I changed that, more clear I think
BUT I found a bug that our existing .env files under /templates were using CR (probably was switched by mistake by somebody) and I switched back to CRLF and now it works fine, I couldn't understand sometime why I have CR in the output :suspect: